library/java/org/chromium/net/impl/CronetUrlRequest.java

```diff
@@ -774,6 +777,121 @@ void onFailed(final UrlResponseInfo urlResponseInfo, final CronetException e) {
     }
   }
 
+  private static String textResponseCode(int code) {
```

Comment:
Would this make sense to be put in a separate file/class?

Reply:
It's a hack - my expectation is this will be short lived - the response message should come from the first line of the HTTP response, example:

```HTTP/1.1 404 Not Found```

As far as I can tell, Envoy Mobile does not expose the first line it receives for the Java layer, so the text had to be "reconstructed". 

I will open an issue for that, or maybe there is somewhere this is available - then I will yank this immediately.